### DISCUSSION TITLE
Is there a way to scroll with my pen on Linux?

### DISCUSSION THREAD

**Discussion by lxhom** [OpenTabletDriver/OpenTabletDriver#1489] [Q&A] [answered]
For example, when i use Firefox or Chrome and use my touchpad to scroll with two fingers, it does smooth scrolling. If I use my pen to scroll it just selects text. The only solution I've found so far to do that with my Wacom CTL-672 is using a 3rd party program to implement mouse gestures and map middle-click + mouse down to scroll down. Shouldn't there be a way to make smooth scrolling with tablets work? It clearly works with touchpads, so it should be possible to do it with a pen, right?

**Comment by adryzz** [comment]
Theoretically it's definitely possible, (i made it work in a PoC a few months ago) but it hasn't been done yet for a plugin.

**Comment by jamesbt365** [comment, marked as answer]
It is doable, but it hasn't been finished yet and thus you cannot do it unfortunately, but it is in the works.

**Reply by Remyueru-AI** [reply]
Any update on this?
